server/athenian/api/models/state/__main__.py

```diff
@@ -23,8 +23,11 @@ def main():
     args = ["alembic", "alembic", "upgrade", "head"]
     if os.getenv("OFFLINE"):
         args.append("--sql")
-    os.execlp(*args)
+    if exec:
+        os.execlp(*args)
+    else:
+        subprocess.run(" ".join(args[1:]), check=True, shell=True)
```

Comment:
Not a big deal, but do we need this? I'd change this as:
```
subprocess.run(args, check=True, capture_output=True)
```
as no need also to add slicing if we remove the first value.

Reply:
`capture_output=True` will not automatically echo the command's output, unfortunately - this is what I wanted here.